hardware/OpenZWave.cpp

```diff
@@ -2300,11 +2300,7 @@ void COpenZWave::AddValue(NodeInfo* pNode, const OpenZWave::ValueID& vID)
 			else if (vOrgIndex == ValueID_Index_Clock::Minute)
 			{
 				pNode->tClockMinute = byteValue;
-				if (
-					(pNode->tClockDay != -1) &&
-					(pNode->tClockHour != -1) &&
-					(pNode->tClockMinute != -1)
-					)
+				if ((pNode->tClockDay != UINT8_MAX) && (pNode->tClockHour != UINT8_MAX) && (pNode->tClockMinute != UINT8_MAX))
```

Comment:
Not a fan of this presentation, the original is much clearer, but would be better if it was
if (
   (a == 1)
   || (b==1)
   || (c==1)
  )

Reply:
Ehhh that can sort of be done with //, but I don't see the reason for it. The conditions are pretty short.